### PR TITLE
Add Katib 2021 ROADMAP

### PR DESCRIPTION
**Pull request by andreyvelich**
Fixes: https://github.com/kubeflow/katib/issues/1484.

This is initial draft for our ROADMAP in 2021.
Please take a look and left your thoughts what we should add/priorities.

/cc @gaocegege @johnugeorge @PatrickXYS @terrytangyuan @kimwnasptd @yanniszark @jbottum @c-bata @RFMVasconcelos @knkski @frankmanbb

### CODE REVIEW

**Review by PatrickXYS — COMMENTED** [review]
/lgtm

Woohoo thanks @andreyvelich  for publishing it

**Review by gaocegege — COMMENTED** [review]
LGTM :+1:
/lgtm

File: ROADMAP.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
...
+- Support Population Based Training [#1382](https://github.com/kubeflow/katib/issues/1382)
+- Support [ASHA](https://arxiv.org/pdf/1810.05934.pdf)
+- Support Auto Model Compression [#460](https://github.com/kubeflow/katib/issues/460)
+- Support Auto Feature Engineering [#475](https://github.com/kubeflow/katib/issues/475)
+- Various CRDs for HP, NAS and other AutoML techniques.
+
+### UI
+
+- Migrate to the new Katib UI [Project 1](https://github.com/kubeflow/katib/projects/1)
+- Hyperparameters visualization with fANOVA algorithm
```
Comment (c-bata) [inline ROADMAP.md:16]: [nits] I'd say "Hyperparameter importances" is more precise. What do you think?

```suggestion
- Hyperparameter importances visualization with fANOVA algorithm
```

**Review by c-bata — COMMENTED** [review]
LGTM with nits!

**Comment by kimwnasptd** [comment]
/lgtm

Looking forward to it :smile:

**Review by terrytangyuan — APPROVED** [review]
Looks great! Thanks for putting this together.

File: ROADMAP.md (hunk 1 above)
Comment (andreyvelich) [reply ROADMAP.md:16]: Sure, thank you!

**Comment by c-bata** [comment]
/lgtm

Looks good!

**Comment by andreyvelich** [comment]
Thanks everyone for the review!
I will remove WIP status and give 4 more days for our users/contributors to check it.
/hold

**Comment by andreyvelich** [comment]
Let's merge this.
/hold cancel

**Pull request merged**